### DISCUSSION TITLE
Wallite Desktop App Using Flet

### DISCUSSION THREAD

**Discussion by LineIndent** [flet-dev/flet#449] [Show and tell]
Hello everyone,

I opened this discussion to showcase an application I built using the Flet library.
It's a card number storage app that allows a user to save a card number (16-digit number) plus bank name and gives them the ability to copy the number (by simply clicking on the text) to their clipboard to easily paste it when needed.

I've done this app using another library, but Flet offers more options with regards to MD and so I decided to update the app a little. 

![Untitled design](https://user-images.githubusercontent.com/84860195/194344539-39831733-16fc-44b1-95be-de0829deb5bb.gif)

I'll upload a video on how to build this on my YT channel (https://www.youtube.com/channel/UC3TYDesNQVGeKldbfL3Rm4Q) when I clear up some bugs and add some more features. 

Finally, many thanks to Feodor and the people who contributed to this awesome library. Thanks!

**Comment by FeodorFitsner** [comment]
That looks really nice!

**Comment by clstaudt** [comment]
Nice work @pourhakimi. Is it open source?

**Reply by LineIndent** [reply]
Thanks @clstaudt . I uploaded the file here https://github.com/pourhakimi/wallite-flet-app.git . I haven't modified much of it, but you're more than welcome to make changes or add to the app.

**Reply by clstaudt** [reply]
Thanks for publishing the code @pourhakimi ! We need more such examples. And most of all we need larger examples of working apps.

I believe I have found a layout but (see issues in your repo).

**Comment by LineIndent** [comment]
I had some extra time to work on this and so I've added some stuff to this app. 
The card details can now be saved to a local database via sqlite. Moreover the card numbers can now be copied to user's system clipboard. 


![Untitled design](https://user-images.githubusercontent.com/84860195/197402401-97f73b26-ca47-4561-ac92-2df50b908b4c.gif)

**Comment by FeodorFitsner** [comment]
You should think about storing sensitive data (CC number) in encrypted form: https://flet.dev/docs/guides/python/encrypting-sensitive-data. User could provide a passphrase on app startup to "unlock" it.
